deploy/roles/neo4j/tasks/main.yml

```diff
@@ -149,6 +149,14 @@
   tags:
   - neo4j
 
+- name: copy neo4j upstart script to /etc/init/neo4j.conf 
+  template: 
+    src: neo4j.upstart.j2 
+    dest: /etc/init/neo4j.conf 
+    mode: 0755
```

Comment:
@olivermel This should be 644 not 755, since it is a configuration file and not an actual binary.